README.md

```diff
@@ -198,6 +155,47 @@ To run all the tests use the rake task, which sets some environment variables fo
 Deployment is through capistrano. To deploy a branch other than "main", prepend an environment variable to your deploy command, e.g.:
 `BRANCH=my_feature bundle exec cap staging deploy`
 
+## When a location changes in Alma:
+
+#### 1. Update the local dev environment:
+
+Load the rails console
+1.`bundle exec rails c`
+
+Delete the existing data from the location tables and repopulate them by pulling data from Alma.
+2.`LocationDataService.delete_existing_and_repopulate`
+
+Generate the following files: marc_to_solr/translation_maps/location_display.rb and marc_to_solr/translation_maps/locations.rb
+3.`LocationMapsGeneratorService.generate`
+
+Copy the content from the generated .rb files into the .tmpl.rb files
+4.`cp marc_to_solr/translation_maps/location_display.rb marc_to_solr/translation_maps/location_display.rb.tmpl`
+`cp marc_to_solr/translation_maps/locations.rb marc_to_solr/translation_maps/locations.rb.tmpl`
+
+Load locally the rails server and make sure that the locations have been updated.
+
+Test the updated locations in Bibdata-staging https://bibdata-staging.princeton.edu/ which is connected to the alma-sandbox; 
+The locations will not be the same as in production because they are not up to date. Deploy your branch on staging and run the following steps to make sure that nothing is breaking the tables.
+
+Connect in one of the bibdata staging boxes:
+1.`ssh deploy@bibdata-alma-staging1`
+2.`cd /opt/marc_liberation/current`
+3.`RAILS_ENV=production bundle exec rails c`
+
+Delete and repopulate the locations in the bibdata staging database:
+4.`LocationDataService.delete_existing_and_repopulate`
```

Comment:
Should we have a rake task to do this?

Reply:
I think having it in a rake task, makes it much easier for someone to run it without thinking what it's actually doing.

Reply:
I was thinking this will make it easier to *automate* the process but I can see how this might not be desirable in this instance.

Reply:
I can add as option 2 what we have in the section to update locations in a new production box. Basically to use capistrano to connect to the db.
`cap [environment] rails:console`
`LocationDataService.delete_existing_and_repopulate`

Reply:
Good idea.